src/components/package-show.js

```diff
@@ -22,100 +20,96 @@ export default function PackageShow({ vendorPackage, packageTitles }, { intl })
-              )}
+          <KeyValueLabel label="Titles Selected">
+            <div data-test-eholdings-package-details-titles-selected>
+              {packageRecord.selectedCount}
+            </div>
+          </KeyValueLabel>
 
-              <hr />
+          <KeyValueLabel label="Total Titles">
+            <div data-test-eholdings-package-details-titles-total>
+              {packageRecord.titleCount}
+            </div>
+          </KeyValueLabel>
 
-              <KeyValueLabel label="Selected">
-                <div data-test-eholdings-package-details-selected>
-                  {packageRecord.isSelected ? 'Yes' : 'No'}
-                </div>
-              </KeyValueLabel>
+          {(packageRecord.customCoverage.beginCoverage || packageRecord.customCoverage.endCoverage) && (
+            <KeyValueLabel label="Custom Coverage">
+              <div data-test-eholdings-package-details-custom-coverage>
+                {formatISODateWithoutTime(packageRecord.customCoverage.beginCoverage)} - {formatISODateWithoutTime(packageRecord.customCoverage.endCoverage)}
+              </div>
+            </KeyValueLabel>
+          )}
 
+          <hr />
+
+          <KeyValueLabel label="Selected">
+            <div data-test-eholdings-package-details-selected>
+              {packageRecord.isSelected ? 'Yes' : 'No'}
+            </div>
+          </KeyValueLabel>
+
+          <hr />
+
+          {packageRecord.visibilityData.isHidden && (
+            <div data-test-eholdings-package-details-is-hidden>
+              <p><strong>This package is hidden.</strong></p>
+              <p><em>{packageRecord.visibilityData.reason}</em></p>
               <hr />
+            </div>
+          )}
 
-              {packageRecord.visibilityData.isHidden && (
-                <div data-test-eholdings-package-details-is-hidden>
-                  <p><strong>This package is hidden.</strong></p>
-                  <p><em>{packageRecord.visibilityData.reason}</em></p>
-                  <hr />
-                </div>
-              )}
-
-              {packageTitles.isResolved && packageTitles.content.length ? (
-                <div>
-                  <h3>Titles</h3>
-                  <List data-test-eholdings-package-details-title-list>
-                    {packageTitles.content.map(item => (
-                      <TitleListItem
-                        key={item.titleId}
-                        item={item}
-                        link={`/eholdings/vendors/${packageRecord.vendorId}/packages/${packageRecord.packageId}/titles/${item.titleId}`}
-                        showSelected
-                      />
-                    ))}
-                  </List>
-                </div>
-              ) : packageTitles.isResolved ? (
-                <p>No Titles Found</p>
-              ) : (
-                <Icon icon="spinner-ellipsis" />
-              )}
+          {packageTitles.isResolved && packageTitles.content.length ? (
```

Comment:
I don't like the blippy-ness of the title flash in sometimes. It feels discombobulating. This isn't for this PR, but maybe we can have some sort of timeout, where we make the requests at the same time, and if they resolve within 1000ms of each other, we render them at the same time, otherwise we show the separate loading spninner.